h_transport_materials/property_database/palladium.py

```diff
@@ -472,6 +472,14 @@
     source="powell_surface_1991",
 )
 
+takagi_recombination_h = RecombinationCoeff(
+    pre_exp=1.5e-27 * 6.022e23 * u.m**4 * u.s**-1 * u.mol**-1,  #convert to mols
+    act_energy=0.48 * 6.022e23 * u.eV * u.mol**-1, #convert to mols
```

Comment:
Thanks to Pint, you don't have to (and shouldn't) convert from mols to particles.

```suggestion
    pre_exp=1.5e-27 * u.m**4 * u.s**-1 * u.particle**-1,
    act_energy=0.48 * u.eV * u.particle**-1,
```